JdeRobot/RoboticsInfrastructure
# cleaned tello_phy driver

To test this:
- Make sure the tello has the latest firmware and the IMU calibrated.
- The equipment connects to the tello correctly and the tello emits a flashing green light.
- Once inside the docker you have to launch the driver:
ros2 launch tello_driver teleop_launch.py 
- in another window with the driver launched try to run ros2 services command like:
ros2 service call /tello_action tello_msgs/TelloAction "{cmd: 'takeoff'}"
ros2 service call /tello_action tello_msgs/TelloAction "{cmd: 'land'}"

In general these messages follow the structure of cmd: 'MESSAGE'.
MESSAGE being any command from the Tello SDK 2.0 command list:
https://dl-cdn.ryzerobotics.com/downloads/Tello/Tello%20SDK%202.0%20User%20Guide.pdf
